committee_admissions/oauth.py

```diff
@@ -64,19 +76,29 @@ def _user_data(self, access_token):
         return self.get_json(url, headers={"AUTHORIZATION": "Bearer %s" % access_token})
 
     def _create_initial_committees(self, access_token):
-        url = urljoin(self.api_url(), "api/v1/groups/?type=komite")
+        url = urljoin(self.api_url(), "api/v1/groups/")
         data = self.get_json(url, headers={"AUTHORIZATION": "Bearer %s" % access_token})
         with transaction.atomic():
             for komite in data["results"]:
                 name = komite["name"]
-                if name == "backup":
+                if name not in self.LEGO_GROUP_NAMES:
```

Comment:
To me it feels kinda counter-intuitive to hardcode certain groups to exclude, instead of just not importing it..

Especially now that the pool of groups is all groups and not only committees

And if we want to make an admin-view where you can create an admission and choose what groups to include down the road a hardcoded exclusion would be annoying af

Aaand for my final reason to prefer removing it, backup now has the option to use this system for admissions if they so desire, and thus it would make even less sense to exclude them